Bio/Phylo/TreeConstruction.py

```diff
@@ -1078,41 +1097,58 @@ class ParsimonyTreeConstructor(TreeConstructor):
 
     Examples
     --------
-    >>> from Bio import AlignIO, Phylo
-    >>> aln = AlignIO.read(open('TreeConstruction/msa.phy'), 'phylip')
-    >>> print(aln)
-    SingleLetterAlphabet() alignment with 5 rows and 13 columns
-    AACGTGGCCACAT Alpha
-    AAGGTCGCCACAC Beta
-    CAGTTCGCCACAA Gamma
-    GAGATTTCCGCCT Delta
-    GAGATCTCCGCCC Epsilon
-    >>> starting_tree = Phylo.read('TreeConstruction/nj.tre', 'newick')
-    >>> print(starting_tree)
-    Tree(rooted=False, weight=1.0)
-        Clade(branch_length=0.0, name='Inner3')
-            Clade(branch_length=0.01421, name='Inner2')
-                Clade(branch_length=0.23927, name='Inner1')
-                    Clade(branch_length=0.08531, name='Epsilon')
+    ::
```

Comment:
Again, rather than ``::`` something introducing the example would be better. e.g. ``We will load an alignment, and then load various trees which have already been computed from it::``